frontend/src/Initialize.tsx

```diff
@@ -1291,6 +1292,18 @@ const Initialize = (props: IProps): JSX.Element => {
           setPlayTakebackPolicy={setPlayTakebackPolicy}
           setBidTakebackPolicy={setBidTakebackPolicy}
         />
+        <div>
+          <label>
+            Game Visibility{" "}
+            <select
+              value={props.state.propagated.game_visibility}
+              onChange={setGameVisibility}
+            >
+              <option value={"Unlisted"}>Unlisted</option>
+              <option value={"Public"}>Public</option>
+            </select>
+          </label>
+        </div>
```

Comment:
I think we also need to update the load/save mechanism (e.g. like https://github.com/rbtying/shengji/blob/master/frontend/src/Initialize.tsx#L1045)